spec/requests/alchemy/json_api/pages_spec.rb

```diff
@@ -193,11 +214,46 @@
 
           it "sets public cache headers of latest published page" do
             get alchemy_json_api.pages_path
-            expect(response.headers["Last-Modified"]).to eq(pages.max_by(&:published_at).published_at.utc.httpdate)
+            expect(response.headers["Last-Modified"]).to be_nil
             expect(response.headers["ETag"]).to match(/W\/".+"/)
             expect(response.headers["Cache-Control"]).to eq("max-age=10800, public, must-revalidate")
           end
 
+          it "returns a different etag if different filters are present" do
+            get alchemy_json_api.pages_path
+            etag = response.headers["ETag"]
+            get alchemy_json_api.pages_path(filter: {page_layout_eq: "standard"})
+            expect(response.headers["ETag"]).not_to eq(etag)
+          end
+
+          it "returns a different etag if different sort params are present" do
+            get alchemy_json_api.pages_path
+            etag = response.headers["ETag"]
+            get alchemy_json_api.pages_path(sort: {id: :asc})
```

Comment:
Sorting [is handled via 
](https://jsonapi.org/format/#fetching-sorting)
```
?sort=id
```

or

```
?sort=-id
```

can we update the spec?


```suggestion
            get alchemy_json_api.pages_path(sort: "id"})
```